README.md

```diff
@@ -111,6 +111,8 @@ Note that `--upload` will only upload the targets that was built and stored in `
 
 You can use `prebuild --upload-all` to upload all files from the `./prebuilds` folder.
 
+You can use `prebuild --upload --tag-prefix <prefix>` for specific tag prefixes for the release. The default prefix is `v` and will result in a tag with an appended version number, for example `v1.0.0`. For [lerna](https://github.com/lerna/lerna) you can use the package name eg `prebuild --tag-prefix "some-package@"` and the binaries will be released on the appropriate package's tags. for example `some-package@1.0.0`.
+
```

Comment:
- The quotes in `--tag-prefix "some-package@"` are not necessary
- `eg` => `e.g.`
- `. for example` => `, for example`